addon-test-support/@ember/test-helpers/helper-hooks.ts

```diff
@@ -1,6 +1,23 @@
-type Hook = (...args: any[]) => void | Promise<void>;
-type HookLabel = 'start' | 'end' | string;
-type HookUnregister = {
+export type Hook = (...args: any[]) => void | Promise<void>;
+export type HookHelperName =
+  | 'blur'
+  | 'click'
+  | 'doubleClick'
+  | 'fillIn'
+  | 'fireEvent'
+  | 'focus'
+  | 'render'
+  | 'scrollTo'
+  | 'select'
+  | 'tab'
+  | 'tap'
+  | 'triggerEvent'
+  | 'triggerKeyEvent'
+  | 'typeIn'
+  | 'visit'
+  | string;
```

Comment:
There are ways we could make this safe, in the future, by *not* allowing `| string`, and by requiring it to be one of the hooks we supply in the library; but that is *definitely* separate from the work you're doing here, so we should just go ahead and remove this entirely.

```suggestion
```